ehealthid-rp/src/main/resources/www/common.css.mustache

```diff
@@ -38,11 +47,12 @@ a {
 
 @media (max-width: 500px) {
   .container {
-    margin: 0 auto;
+    height: 100%;
     border: none;
   }
 
   html body {
+    height: 100svh;
```

Comment:
Just a note: 
`svh` is a recent css unit. If it's not supported in some browser, it will be ignored and `100vh` from `html body { }` will be used as fallback.